File: cmd/sharpfireplace/LogEntryDetailItem.cs
namespace sharpfireplace
{
	public class LogEntryDetailItem
	{
		public string Key { get; set; }
		public string Value { get; set; }

		public LogEntryDetailItem()
		{
			this.Key = "";
			this.Value = "";
		}

		public LogEntryDetailItem(string key, string value)
		{
			this.Key = key;
			this.Value = value;
		}
	}
}

File: pkg/LogEntryDetailItem.go
/*
 * Copyright (c) 2021. App Nerds LLC. All rights reserved
 */

package pkg

/*
LogEntryDetailItem is a key/value pair
*/
type LogEntryDetailItem struct {
	Key   string `json:"key" bson:"key"`
	Value string `json:"value" bson:"value"`
}

/*
LogEntryDetailItemCollection is a collection of key/value pairs.
*/
type LogEntryDetailItemCollection []*LogEntryDetailItem
